src/network/connectionthreads.cpp

```diff
@@ -1157,10 +1157,13 @@ SharedBuffer<u8> ConnectionReceiveThread::handlePacketType_Control(Channel *chan
 				// rtt miscalculation we handle it here
 				float rtt = 0.0f;
 				if (current_time > p.absolute_send_time) {
-					rtt = (current_time - p.absolute_send_time) / 1000.0f;
+					float rtt = (current_time - p.absolute_send_time) / 1000.0;
+
+					// Let peer calculate stuff according to it
+					// (avg_rtt and resend_timeout)
+					dynamic_cast<UDPPeer *>(peer)->reportRTT(rtt);
 				} else if (p.totaltime > 0) {
```

Comment:
The "{" at the end of this line shall go away as its closing "}" was obliterated by this revert.

Reply:
Done